Gems/Atom/Feature/Common/Code/Source/ACES/AcesDisplayMapperFeatureProcessor.cpp

```diff
@@ -56,395 +57,409 @@ namespace
     }
 }
 
-namespace AZ
+namespace AZ::Render
 {
-    namespace Render
+    void AcesDisplayMapperFeatureProcessor::Reflect(ReflectContext* context)
     {
-        void AcesDisplayMapperFeatureProcessor::Reflect(ReflectContext* context)
+        if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
         {
-            if (auto* serializeContext = azrtti_cast<SerializeContext*>(context))
-            {
-                serializeContext
-                    ->Class<AcesDisplayMapperFeatureProcessor, FeatureProcessor>()
-                    ->Version(0);
-            }
+            serializeContext
+                ->Class<AcesDisplayMapperFeatureProcessor, FeatureProcessor>()
+                ->Version(0);
         }
+    }
 
-        void AcesDisplayMapperFeatureProcessor::Activate()
-        {
-            GetDefaultDisplayMapperConfiguration(m_displayMapperConfiguration);
-        }
+    void AcesDisplayMapperFeatureProcessor::Activate()
+    {
+        GetDefaultDisplayMapperConfiguration(m_displayMapperConfiguration);
+    }
+
+    void AcesDisplayMapperFeatureProcessor::Deactivate()
+    {
+        m_ownedLuts.clear();
+    }
+
+    void AcesDisplayMapperFeatureProcessor::Simulate(const FeatureProcessor::SimulatePacket& packet)
+    {
+        AZ_TRACE_METHOD();
```

Comment:
What's the trace here for?

Reply:
AcesDisplayMapperFeatureProcessor didn't actually change much, but I changed indentation when I collapsed the namespace so github gets pretty confused with the diff. So I didn't actually touch any of this particular code. It looks kinda silly for this to be here, but maybe some profiler somewhere is expecting to at least see simulate getting called on every feature processor? I couldn't really remove without testing.